building/devices/rpi3.rst

```diff
@@ -330,25 +329,61 @@ After this, restart the NFS kernel server
 .. hint::
 
     To see that your shares are correctly setup and that the NFS server is
-    running, you can run: ``$ showmount --all localhost`` and you should get a
-    list of ``IP:<path>'s`` based on what you have added in your exports file.
-    If you get nothing there, then your NFS server hasn't been setup correctly.
+    running, you can run: ``$ showmount --exports localhost`` and you should
+    get a list of folders exported based on what you have added in your exports
+    file. If you get nothing there, then your NFS server hasn't been setup
+    correctly.
 
 Prepare files to be shared
 ==========================
-We are now going to put the root filesystem on the location we prepared in the
-previous section.
+In the following step, we'll place the root filesystem in the location prepared
+earlier. To accomplish this, we must first extract the rootfs partition from
+the image generated during the build process for our device. Let's start by
+finding the offset to the rootfs image:
 
 .. note::
 
-    The path to the ``rootfs.cpio.gz`` refers to <rpi3-project>, replace this so
-    it matches your setup.
+    The path to the ``rpi3-sdcard.img`` refers to <rpi3-project>, replace this
+    so it matches your setup.
+
+.. code-block:: bash
+
+    # Find the offset to the rootfs image
+    $ fdisk -l <rpi3-project>/out/rpi3-sdcard.img
+
+    # Which will output something like this:
+    Device               Boot  Start    End Sectors  Size Id Type
+    .../rpi3-sdcard.img1 *      2048 131071  129024   63M  e W95 FAT16 (LBA)
+    .../rpi3-sdcard.img2      133120 657407  524288  256M 83 Linux
+
+Now we need to loopback mount the image with the offset in to the
+``rpi3-sdcard.img2`` which is the partition for the root filesystem. The offset
+is found by taking the ``sector size * start offset``. The ``sector size`` on
+our build is ``512`` (can be found by ``Size / Sectors``, i.e., ``256*1024*1024
+/ 524288`` from the example above). Once we know this, we can loopback mount
+the image.
+
+.. code-block:: bash
+
+   $ sudo mount -o loop,offset=$((512*133120)) <rpi3-project>/out/rpi3-sdcard.img /mnt
```

Comment:
Or use `losetup` to avoid the offset calculation. Something like:
```
sudo losetup -Pf <rpi3-project>/out/rpi3-sdcard.img
sudo mount /dev/loop0p1 /mnt
...
sudo umount /mnt
sudo losetup -d /dev/loop0
```

Reply:
Much better, I will update the PR according, thanks!